components/browser/session/src/main/java/mozilla/components/browser/session/Session.kt

```diff
@@ -40,6 +42,8 @@ class Session(
         fun onSecurityChanged(session: Session, securityInfo: SecurityInfo) = Unit
         fun onCustomTabConfigChanged(session: Session, customTabConfig: CustomTabConfig?) = Unit
         fun onDownload(session: Session, download: Download): Boolean = false
+        fun onTrackerBlockingEnabledChanged(session: Session, blockingEnabled: Boolean): Unit = Unit
+        fun onTrackerBlocked(session: Session, blocked: String, all: List<String>): Unit = Unit
```

Comment:
\[ktlint\] WARNING: \[no\-unit\-return\] Unnecessary "Unit" return type in column 85